lib/api/2.0/views.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2016, EMC Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var views = injector.get('Views');
+var Promise = injector.get('Promise');
```

Comment:
This warning can be ignored, since we have already pre-defined the `Promise`, `_` and some other variables in jshint config file.